tests/Settings/TestSettingVisibilityPresets.py

```diff
@@ -0,0 +1,90 @@
+from unittest.mock import MagicMock
+
+from UM.Preferences import Preferences
+import os.path
+
+from UM.Preferences import Preferences
+from UM.Resources import Resources
+from cura.CuraApplication import CuraApplication
+from cura.Machines.Models.SettingVisibilityPresetsModel import SettingVisibilityPresetsModel
+from cura.Settings.SettingVisibilityPreset import SettingVisibilityPreset
+
+setting_visibility_preset_test_settings = set(["test", "zomg", "derp", "yay", "whoo"])
+
+Resources.addSearchPath(os.path.abspath(os.path.join(os.path.join(os.path.dirname(__file__)), "../..", "resources")))
```

Comment:
This won't work on Windows because you're hard-coding a forward slash for one of the path separators. Also, the nested `os.path.join` call is unnecessary. I'd suggest this:

    Resources.addSearchPath(os.path.abspath(os.path.join(os.path.dirname(__file__), "..", "..", "resources")))